pkg/asset/agent/installconfig_test.go

```diff
@@ -4,11 +4,12 @@ import (
 	"net"
 	"testing"
 
-	"github.com/golang/mock/gomock"
-	"github.com/stretchr/testify/assert"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/pointer"
 
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
```

Comment:
Judging by the multiple changes to imports like this, I think your formatting tools are configured in a way that's not going to be compatible with the installer linter. The linter will want all of these external packages to be grouped in a single block.